tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -5,12 +5,7 @@ test.describe( 'Nested Tabs tests', () => {
 	test( 'Count the number of icons inside the Add Section element', async ( { page }, testInfo ) => {
 		// Arrange.
 		const wpAdmin = new WpAdminPage( page, testInfo );
-		await wpAdmin.setExperiments( {
-			container: true,
-			'nested-elements': true,
-		} );
-
-		const editor = await wpAdmin.useElementorCleanPost(),
+		const editor = await wpAdmin.openNewPage(),
```

Comment:
Why `openNewPage()` and not `useElementorCleanPost()`?

Reply:
Done